callbacks.php

```diff
@@ -59,7 +55,7 @@
 }
 
 // Verify that callback is genuine. Exit if not.
-if ($expectedsecret !== $reqheaders['X-Turnitin-Signature']) {
+if ($expectedsecret !== $reqheaders['x-turnitin-signature']) {
```

Comment:
If it is easy can we please write unit test cases?

Reply:
I looked into that but didn't see how this file is testable without a rewrite/big refactor since it's just a PHP script and not using classes/methods.